### ISSUE TITLE
Getting error while generating looper build with underscore

### ISSUE THREAD

**Issue by TharaCJ** [jashkenas/underscore#2839]
In the code we are setting templateSettings object of underscore as shown below

_._templateSettings = {
        interpolate: /\{\{(.+?)\}\}/g
    };
Upon doing a local build and launching the application, it works fine.But when the build is generated using looper and deployed to any environment, the application is not launching and in console we see the error - Cannot set property of #<Object> which has only a getter

Appreciate any help to resolve this issue.

**Comment by jgonggrijp** [comment]
I don't know looper so I'm guessing a bit. It seems like your build tool is freezing the Underscore exports. If you are doing this:

```js
import * as _ from 'underscore';
```

you can try replacing it by this:

```js
import _ from 'underscore';
```

Otherwise, please give us a bit more information about your build process and the way in which you import Underscore.

**Comment by TharaCJ** [comment]
Yes , Looper is the build tool.
The application is in angular1.x so this is how we are adding the dependency

define(['underscore', 'jquery'], function(_, $){
    'use strict';

    _.templateSettings = {
        interpolate: /\{\{(.+?)\}\}/g
    };

**Comment by jgonggrijp** [comment]
I tried searching the web but I don't think I have found the right Looper yet. Could you give me a link to its homepage? And also show your command line that invokes Looper in order to build?

**Comment by demetris-manikas** [comment]
Hi. I am getting exactly the same error when using webpack-dev-server.
No messages or error when I deploy normally.
Are you familiar with webpack-dev-server or should I prepare a showcase project?

**Comment by jgonggrijp** [comment]
I know it a little but I'd appreciate a showcase project very much!

One disclaimer in advance: the webpack dev server has a really exotic way of bundling the code. Among other things, it breaks sourcemaps in Safari. I'll try my best, though.

**Comment by demetris-manikas** [comment]
I will either come up with a showcase or a PR very soon.

**Comment by demetris-manikas** [comment]
Well... I tried my best and I came up to the conclusion that this is maybe unsolvable the way me and @TharaCJ expected it to work. 
BUT!
I have good news since I came up with a workaround that is not ugly at all!
Try  `_.templateSettings.interpolate =  /\{\{(.+?)\}\}/g;`
It worked for my case.  Tell us if it worked for you as well;
@jgonggrijp I am fine with this solution so I wont pursue this any more.
